cypress/e2e/cloud/globalHeader.ts

```diff
@@ -0,0 +1,59 @@
+describe('multi-account multi-org global header', () => {
+  const globalHeaderFeatureFlags = {
+    quartzIdentity: true,
+    multiOrg: true,
+  }
+
+  beforeEach(() => {
+    // Maintain the same session for all tests so that further logins aren't required.
+    Cypress.Cookies.preserveOnce('sid')
```

Comment:
This is a brilliant idea, i think it might save us a lot of time on other tests where re-logins are taking 10+seconds each run, and are not necessary.